security/fma-holocene.md

```diff
@@ -0,0 +1,117 @@
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Holocene Derivation](#holocene-derivation)
+  - [The batcher violates the new (stricter) ordering rules](#the-batcher-violates-the-new-stricter-ordering-rules)
+    - [Description](#description)
+    - [Risk Assessment](#risk-assessment)
+    - [Mitigations](#mitigations)
+    - [Detection](#detection)
+    - [Recovery Path(s)](#recovery-paths)
+  - [Derivation Deadlock (either specification or op-node bug)](#derivation-deadlock-either-specification-or-op-node-bug)
+    - [Description](#description-1)
+    - [Risk Assessment](#risk-assessment-1)
+    - [Mitigations](#mitigations-1)
+    - [Detection](#detection-1)
+    - [Recovery Path(s):](#recovery-paths)
+- [Deterministic Standard L2 Genesis](#deterministic-standard-l2-genesis)
+- [Configurable EIP-1559 Parameters via SystemConfig](#configurable-eip-1559-parameters-via-systemconfig)
+- [L2ToL1MessagePasser Storage Root in Header](#l2tol1messagepasser-storage-root-in-header)
+- [Update to the MIPS contract](#update-to-the-mips-contract)
+- [Generic Items](#generic-items)
+- [Audit Requirements](#audit-requirements)
+- [Action Items](#action-items)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+| | |
+|--------|--------------|
+| Author | George Knee |
+| Created at | 2024-09-24 |
+| Needs Approval From | maurelian |
+| Other Reviewers |   |
+| Status | Draft |
+
+
+## Introduction
+
+This document is intended to be shared in a public space for reviews and visibility purposes. It covers the Holocene hardfork, which involves the following changes: 
+- (Consensus Layer) Holocene Derivation
+- (Smart Contracts) Deterministic Standard L2 Genesis
+- (Execution Layer & Smart Contracts) Configurable EIP-1559 Parameters via SystemConfig
+- (Execution Layer) L2ToL1MessagePasser Storage Root in Header
+- (Smart Contracts) Update to the MIPS contract
+
+Each change has it's own section below with a list of Failure Modes.
+
+Below are references for this project:
+
+- [PID: Holocene hardfork upgrade](https://www.notion.so/PID-Holocene-hardfork-upgrade-00ee1ffc414a407088fdb49841771527?pvs=21)
+- [Github tracker](https://github.com/orgs/ethereum-optimism/projects/84/views/6)
+- [Specs](https://specs.optimism.io/protocol/holocene/derivation.html?highlight=holocene#holocene-derivation)
+
+
+##  Holocene Derivation
+
+### The batcher violates the new (stricter) ordering rules
+
+#### Description
+
+If the batcher:
+  - sends transactions out of order, or
+   - orders frames within a transaction out of order
+
+This will lead to the consensus client hitting error conditions when it loads the frames from L1. According to the spec, frames will be dropped in order to maintain a contiguous frame queue. If those frames are not re-sent in a timely manner on L1, the safe chain would halt.
+    
+#### Risk Assessment
+ medium severity / medium likelihood
```

Comment:
imo "low severity" (restart fixes, short safe chain halts are non-critical) and "low likelihood" (it already almost never happens in prod with the current impl, and we're already aware of the specific edge cases that need hardening)